packages/scandipwa/src/component/Field/Field.component.js

```diff
@@ -294,6 +294,7 @@ export class Field extends PureComponent {
     // Renders fields label above field
     renderLabel() {
         const { type, label, attr: { name } = {} } = this.props;
+        console.log(label);
```

Comment:
Please check the code before you make a commit and remove all debugging code, commented out code, etc.